pkg/workflows/create.go

```diff
@@ -349,8 +349,8 @@ func (s *DeleteBootstrapClusterTask) Name() string {
 
 func getManagementCluster(commandContext *task.CommandContext) *types.Cluster {
```

Comment:
I there a way we can avoid having this method and just use grab `commandContext.ManagementCluster`?

Reply:
This is only used once where we need the eksa management cluster for the ApplyBundles method. Can just substitute the if else here though.

Reply:
Oh nvm, I get it now, this is cool with me